microcosm_flask/sync/push.py

```diff
@@ -91,7 +91,10 @@ def iter_json_batches(inputs, base_url, batch_size):
             yield (uri, [resource])
         else:
             # batch handling
-            collection_uri = uri.rsplit("/", 1)[0]
+            if keep_instance_path:
```

Comment:
I wonder if we can find a way to express the batch spilt function on the CLI instead of having a conditional.